src/webapp/js/components/modules-form.js

```diff
@@ -250,9 +248,6 @@ class ModuleForm extends Component {
 
     if (attributes && attributes.type === 'checkbox') {
       attributes.checked = input.value;
-      attributes.onChange = (event) => {
```

Comment:
@dserkowski why did you removed this?

Reply:
I noticed that when we have many Choices and we select or diselect one of their choiceboxes, it causes not only the change of the selected box value, but also the change of other choiceboxes values.